ams/server/src/main/java/com/netease/arctic/server/table/executor/OptimizingExpiringExecutor.java

```diff
@@ -28,20 +28,19 @@
 public class OptimizingExpiringExecutor extends BaseTableExecutor {
   private static final Logger LOG = LoggerFactory.getLogger(OptimizingExpiringExecutor.class);
 
-  // 1 days
-  private static final long INTERVAL = 24 * 60 * 60 * 1000L;
-  // 30 days
-  private static final long KEEP_TIME = 30 * 24 * 60 * 60 * 1000L;
-
   private final Persistency persistency = new Persistency();
+  private final long keepTime;
+  private final long interval;
 
-  public OptimizingExpiringExecutor(TableManager tableRuntimes) {
+  public OptimizingExpiringExecutor(TableManager tableRuntimes, int keepDays, int intervalHours) {
     super(tableRuntimes, 1);
+    this.keepTime = keepDays * 24 * 60 * 60 * 1000L;
+    this.interval = intervalHours * 60 * 60 * 1000L;
```

Comment:
Use `Duration.ofDays().toMillis()` and `Duration.ofHours().toMillis()` instead may be better for read.

Reply:
I saw `data-expiration.interval` is configure (`1d`) and I thought we should use the same style.